app/code/core/Mage/Api2/Model/Resource/Validator/Fields.php

```diff
@@ -58,7 +58,7 @@ class Mage_Api2_Model_Resource_Validator_Fields extends Mage_Api2_Model_Resource
      *
      * @var array
      */
-    protected $_requiredFields;
+    protected $_requiredFields = array();
```

Comment:
`= []` 😸

Reply:
I thought we'll keep the old syntax until your PR is merged and this is enforced with CI 😅